modules/oe_content_news/oe_content_news.install

```diff
@@ -9,6 +9,17 @@ declare(strict_types = 1);
 
 use Drupal\Core\Entity\Entity\EntityViewDisplay;
 
+/**
+ * Implements hook_install().
+ */
+function oe_content_news_install() {
+  // Install the newly defined SKOS Concept defined fields.
+  /** @var \Drupal\rdf_skos\SkosEntityDefinitionUpdateManager $update_manager */
```

Comment:
We need to limit at least the definition installation to the fields defined by this very module.